pages/linux/toolbox-init-container.md

```diff
@@ -0,0 +1,9 @@
+# toolbox init-container
+
+> Initialize a running `toolbox` container.
+> This command is not expected to be directly invoked by humans, and cannot be used on the host.
+> More information: <https://manned.org/toolbox-init-container.1>.
+
+- Initialize a running toolbox:
+
+`toolbox init-container --gid {{gid}} --home {{home}} --home-link --media-link --mnt-link --monitor-host --shell {{shell}} --uid {{uid}} --user {{user}}`
```

Comment:
Are the `--...-link` and `-monitor-host` arguments required? If not you could maybe make another example, so that the first one is as minimal as it can be. (But only if it makes sense without them).

Reply:
I haven't used init containers before. I referred to this from the synopsis at <https://github.com/containers/toolbox/blob/main/doc/toolbox-init-container.1.md> ,

Reply:
@marchersimon maybe we can skip this page. I will remove it if necessary. It seems a bit confusing.

Reply:
I think we can keep the page. At least we mention that it's not supposed to be used by humans.

Regarding the rename to `toolbx`, usually we add an alias page for this. However, in the blog they state that this alias doesn't exist yet:

> So, those will gradually happen over time with all the necessary compatibility aliases and such.

You could add an alias page once the `toolbx` binary exists if you want to.

Reply:
Sure, then I will leave this page as it is. In future when it becomes toolbx I will add an alias page.